packages/alsatian/core/alsatian-core.ts

```diff
@@ -71,6 +71,7 @@ export {
 	AsyncTeardownFixture,
 	AsyncTest,
 	buildExpect,
+	CallbackTestRunner,
```

Comment:
My idea was that you could create own runners for specific needs. Like my runner works different it reports test executions  to some webstorm HTTP-REST Interface instead of parsing the system out. I was not sure you would allow me to add my webstorm runner to your library. As long as i can use it somehow it will work :)